tests/test_sraweb.py

```diff
@@ -14,3 +14,138 @@ def sraweb_connection():
 def test_sra_metadata(sraweb_connection):
     df = sraweb_connection.sra_metadata("SRP016501")
     assert df.shape[0] == 134
+
+
+def test_tissue_column(sraweb_connection):
```

Comment:
Can you add one line description to each method such as """Test if tissue column exists""".